OpenSSL/README.md

```diff
@@ -0,0 +1,33 @@
+# Build OpenSSL as static PIE
+
+Script to build OpenSSL as a static PIE binary for use with Unikraft
```

Comment:
You shouldn't mention using the app over `Unikraft`.
Use the template from [nginx](https://github.com/unikraft/static-pie-apps/blob/master/nginx/README.md?plain=1#L3), it should say something along the lines of "Build `OpenSSL` as a static PIE ELF".